end-user-surveys/otel-collector/README.md

```diff
@@ -0,0 +1,12 @@
+# OTel Collector Survey
+
+## Purpose
+
+To help the OTel Collector SIG better understand the usage of the OTel Collector, so that they can develop a roadmap that best suits users' needs.
+
+## Resources
+
+* [CSV Survey Results](/end-user-surveys/otel-collector/otel-collector-survey.csv)
```

Comment:
* CSV survey results
* PDF survey results
* Blog summary of survey results
* User feedback discussion panel